metadata/contributors.json

```diff
@@ -1550,6 +1550,13 @@
       ],
       "status" : "committer"
    },
+   {
+      "emails" : [
+         "cgibb2@apple.com"
+      ],
+      "name" : "Chris Gibb",
+      "status" : "committer"
```

Comment:
You need to add your GitHub username too, otherwise you won't be able to add labels (and EWS doesn't actually know that `chgibb-apple` and `Chris Gibb` are the same person, unless we tell it)

Reply:
Thank you. Fixed!